prov/efa/src/rdm/efa_rdm_ope.c

```diff
@@ -167,6 +175,14 @@ void efa_rdm_rxe_release_internal(struct efa_rdm_ope *rxe)
 
 	dlist_remove(&rxe->ep_entry);
 
+	/**
+	 * Make sure the entry is removed
+	 * from ope_longcts_list when the ope
+	 * is released for whatever reasons.
+	 */
+	if (rxe->state == EFA_RDM_TXE_SEND)
```

Comment:
Is rxe->state really equal to `EFA_RDM_TXE_SEND`?  Why?

Reply:
Yes, the macro is a bit misleading, because we can post a send from an rxe as well https://github.com/ofiwg/libfabric/blob/main/prov/efa/src/rdm/efa_rdm_pke_nonreq.c#L411. Lemme rename it to `EFA_RDM_OPE_SEND`